js/controller.js

```diff
@@ -146,6 +151,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       $("#" + elementId + " .player_skin").css("z-index", OO.CSS.ALICE_SKIN_Z_INDEX);
       this.state.mainVideoElement = $("#" + elementId + " .video");
 
+      this.state.playerHeight = $("#" + elementId + " .innerWrapper").height();
```

Comment:
Can we do this without direct DOM manipulation (jQuery) and stick to React's virtual DOM?

It's considered best practice to not mix direct DOM manipulation with React.

See this for more details - https://facebook.github.io/react/docs/working-with-the-browser.html